shell-completion/bash/networkctl

```diff
@@ -32,7 +32,7 @@ _networkctl() {
     local i verb comps
     local cur=${COMP_WORDS[COMP_CWORD]} prev=${COMP_WORDS[COMP_CWORD-1]}
     local -A OPTS=(
-        [STANDALONE]='-a --all -h --help --version --no-pager --no-legend'
+        [STANDALONE]='-a --all -h --help --version --no-pager --no-legend -s --stats'
```

Comment:
Would you mind doing zsh too?

Reply:
Sorry, I've never used zsh, and am not familiar with zsh-completion...